install-binary.sh

```diff
@@ -1,23 +1,21 @@
-#!/usr/bin/env bash
+#!/usr/bin/sh sh
 
 # Shamelessly copied from https://github.com/technosophos/helm-template
 
 PROJECT_NAME="helm-diff"
 PROJECT_GH="databus23/$PROJECT_NAME"
-GREP_COLOR="never"
-
-: ${HELM_PLUGIN_DIR:="$(helm home --debug=false)/plugins/helm-diff"}
+export GREP_COLOR="never"
 
 # Convert the HELM_PLUGIN_DIR to unix if cygpath is
 # available. This is the case when using MSYS2 or Cygwin
 # on Windows where helm returns a Windows path but we
 # need a Unix path
 
-if type cygpath > /dev/null 2>&1; then
+if type cygpath >/dev/null 2>&1; then
```

Comment:
Is this a cosmetic change, or actually required for POSIX compliance?

Reply:
Cosmetic change